src/braket/experimental/autoqasm/instructions/gates.py

```diff
@@ -29,6 +29,7 @@ def ccnot(
     control_0: QubitIdentifierType,
     control_1: QubitIdentifierType,
     target: QubitIdentifierType,
+    **kwargs,
```

Comment:
I chose to add `**kwargs` to each gate here, rather than listing out the `control`, `control_state`, and `power` args for each gate. The upside is code brevity and less redundancy. The downside is that the args are not directly documented for each gate. Interested if anyone has a strong opinion either way.

Reply:
+1 I think it's okay for now. Can always be updated. Do you mind adding a docstring for `_qubit_instruction` and perhaps reference this trade off? It could help developers to understand why the function is frequently called with kwargs